test/data/testdata.txt

```diff
@@ -0,0 +1 @@
+224e6f7468696e6720746f20646f3f22aa4e6f7468696e6720746f20646f3f204e6f7468696e6720746f20646f3fa50757420736f6d65206d75737461726420696e20796f75722073686f652ca46696c6c20796f757220706f636b6574732066756c6c206f6620736f6f742ca44726976652061206e61696c20696e746f20796f757220666f6f742ca50757420736f6d6520737567617220696e20796f757220686169722ca506c61636520796f757220746f79732075706f6e207468652073746169722ca536d65617220736f6d65206a656c6c79206f6e20746865206c617463682ca45617420736f6d65206d756420616e6420737472696b652061206d617463682ca4472617720612070696374757265206f6e207468652077616c6c2ca526f6c6c20736f6d65206d6172626c657320646f776e207468652068616c6c2ca506f757220736f6d6520696e6b20696e2044616464792773206361702d2d2da4e6f7720676f20757073746169727320616e642074616b652061206e61702eaa2020202020202020206279205368656c6c65792053696c766572737465696e
```

Comment:
I want to make it automated. So that we don't have to create test files. Just run one command and that's it.